vs2017/libtundra/libtundra.vcxproj

```diff
@@ -1,181 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Removal of VS projects -- what's the preferred way of working with Tundra codebase then now? Does Bee build of it generate the VS projects?

Reply:
ugh :)  ok, i've added support for this and pushed it.  ps, our native visualstudio support is really poor